planner/core/physical_plans.go

```diff
@@ -791,6 +791,9 @@ type PhysicalTableScan struct {
 	// AccessCondition is used to calculate range.
 	AccessCondition []expression.Expression
 	filterCondition []expression.Expression
+	// pushedDownFilterCondition is used to record the filter conditions
+	// that are pushed down to table scan from selection.
+	pushedDownFilterCondition []expression.Expression
```

Comment:
maybe `tableScanFilterCondition` a better name

Reply:
No clear, the name of struct is `PhysicalTableScan` which contain `tablescan`, so we may confusing about `tableScanFilterCondition` and `filterCondition`

Reply:
Calling the pushdown filter condition directly will be a bit problematic, and it needs to be able to highlight that these predicates are predicates that are delayed materialization.
Because at the execution layer, all filters on selection will be used as push_down_filter in PhysicalTableScan. Such a naming would cause ambiguity.

Reply:
rename to `prewhereFilterCondition`